src/views/TableView.js

```diff
@@ -80,9 +80,11 @@ function TableView(props, { direction }) {
     setOpen(false);
   };
 
+  let num = 4; // num is the number of columns visible by default
+  // The columns visible by default are the columns: 1 to num
   // Columns that are selected visible in table
   const [visibleColumns, setVisibleColumns] = useState(
-    columns.map((c) => c.key)
+    columns.map((c) => c.key).slice(0, num)
```

Comment:
This solution assumes that the columns array is sorted in such an order that the first four objects are those that we want to display by default. If someone would we would want to change this array (order or elements or something), then the default parameters could possibly be (unintentionally) affected as a side-effect.

I'd recommend to refactor this to use an array of the keys in the same file as the columns are defined (TableViewColumns.js). As the table columns array contains objects with a key-field like so:

```
export const TableViewColumns = [
  {
    key: resultRowsEnums.POINT_INDEX,
    name: "index",
    resizable: true,
    sortable: true,
  },
  {
    key: resultRowsEnums.GDO_GID,
    name: "GDO_GID",
    resizable: true,
    sortable: true,
  },
  ...
]
```

The keys could be used as entries in the default column array like so:

```
export const defaultTableViewColumns = [
  resultRowsEnums.POINT_INDEX,
  resultRowsEnums.GDO_GID,
  resultRowsEnums.VAYLAT,
  resultRowsEnums.RISK_INDEX_SUM
]
```
Then the default columns wouldn't be affected by changing the columns array. The default columns can be selected like so:

`columns.map((c) => c.key).filter((c) => defaultTableViewColumns.includes(c.key))`